Jellyfin.Api/Controllers/StartupController.cs

```diff
@@ -73,43 +83,48 @@ public StartupConfigurationDto GetStartupConfiguration()
             _config.Configuration.MetadataCountryCode = metadataCountryCode;
             _config.Configuration.PreferredMetadataLanguage = preferredMetadataLanguage;
             _config.SaveConfiguration();
+            return Ok();
         }
 
         /// <summary>
-        /// Endpoint for (dis)allowing remote access and UPnP.
+        /// Sets remote access and UPnP.
         /// </summary>
         /// <param name="enableRemoteAccess">Enable remote access.</param>
         /// <param name="enableAutomaticPortMapping">Enable UPnP.</param>
+        /// <response code="200">Configuration saved.</response>
+        /// <returns>Status.</returns>
         [HttpPost("RemoteAccess")]
-        public void SetRemoteAccess([FromForm] bool enableRemoteAccess, [FromForm] bool enableAutomaticPortMapping)
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public ActionResult SetRemoteAccess([FromForm] bool enableRemoteAccess, [FromForm] bool enableAutomaticPortMapping)
         {
             _config.Configuration.EnableRemoteAccess = enableRemoteAccess;
             _config.Configuration.EnableUPnP = enableAutomaticPortMapping;
             _config.SaveConfiguration();
+            return Ok();
         }
 
         /// <summary>
-        /// Endpoint for returning the first user.
+        /// Gets the first user.
         /// </summary>
+        /// <response code="200">Initial user retrieved.</response>
         /// <returns>The first user.</returns>
         [HttpGet("User")]
-        public StartupUserDto GetFirstUser()
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public ActionResult<StartupUserDto> GetFirstUser()
         {
             var user = _userManager.Users.First();
-            return new StartupUserDto
-            {
-                Name = user.Name,
-                Password = user.Password
-            };
+            return new StartupUserDto { Name = user.Name, Password = user.Password };
         }
 
         /// <summary>
-        /// Endpoint for updating the user name and password.
+        /// Sets the user name and password.
         /// </summary>
         /// <param name="startupUserDto">The DTO containing username and password.</param>
+        /// <response code="200">Updated user name and password.</response>
         /// <returns>The async task.</returns>
```

Comment:
Maybe this is a bit verbose, but I think the result object contained by the task is probably more important to document than the task itself

```suggestion
        /// <returns>
        /// A <see cref"Task" /> that represents the asynchronous update operation.
        /// The task result contains an <see cref="OkResult"/> indicating success.
        /// </returns>
```

Reply:
I think just stating the object returned within the task is enough, but we can also standardize the docs being extra verbose if that's desired

Reply:
My preference would be to be more verbose, since in this case being more verbose also more accurately describes what is being returned